packages/Interface/src/DTK_Fortran_API.f90

```diff
@@ -0,0 +1,169 @@
+! This file was automatically generated by SWIG (http://www.swig.org).
+! Version 4.0.0
+!
+! Do not make changes to this file unless you know what you are doing--modify
+! the SWIG interface file instead.
+module DataTransferKit
+ use, intrinsic :: ISO_C_BINDING
+ implicit none
+
+ ! PUBLIC METHODS AND TYPES
+ public :: DTK_version
+ public :: DTK_git_commit_hash
+ public :: DTK_ExecutionSpace, DTK_SERIAL, DTK_OPENMP, DTK_CUDA
+ public :: DTK_create
+ public :: DTK_is_valid
+ public :: DTK_destroy
+ public :: DTK_initialize
+ public :: DTK_is_initialized
+ public :: DTK_finalize
+ public :: DTK_FunctionType, DTK_NODE_LIST_SIZE_FUNCTION, DTK_NODE_LIST_DATA_FUNCTION, DTK_BOUNDING_VOLUME_LIST_SIZE_FUNCTION, &
+    DTK_BOUNDING_VOLUME_LIST_DATA_FUNCTION, DTK_POLYHEDRON_LIST_SIZE_FUNCTION, DTK_POLYHEDRON_LIST_DATA_FUNCTION, DTK_CELL_LIST_SIZE_FUNCTION, &
+    DTK_CELL_LIST_DATA_FUNCTION, DTK_MIXED_TOPOLOGY_CELL_LIST_SIZE_FUNCTION, DTK_MIXED_TOPOLOGY_CELL_LIST_DATA_FUNCTION, DTK_BOUNDARY_SIZE_FUNCTION, &
+    DTK_BOUNDARY_DATA_FUNCTION, DTK_DOF_MAP_SIZE_FUNCTION, DTK_DOF_MAP_DATA_FUNCTION, DTK_MIXED_TOPOLOGY_DOF_MAP_SIZE_FUNCTION, &
+    DTK_MIXED_TOPOLOGY_DOF_MAP_DATA_FUNCTION, DTK_FIELD_SIZE_FUNCTION, DTK_PULL_FIELD_DATA_FUNCTION, DTK_PUSH_FIELD_DATA_FUNCTION, DTK_EVALUATE_FIELD_FUNCTION
+ public :: DTK_set_function
+ ! TYPES
+ enum, bind(c)
+  enumerator :: DTK_ExecutionSpace = -1
```

Comment:
Not sure. This is the way SWIG/Fortran generated them. I'll have to ask Seth about that.